runtime/gc_modron_startup/mminit.cpp

```diff
@@ -2724,6 +2725,9 @@ gcInitializeDefaults(J9JavaVM* vm)
 			/* running interpreted is ok on any h/w */
 			extensions->concurrentScavenger = true;
 		}
+#endif
+		/* s390 JIT supports software read barriers */
+		extensions->concurrentScavenger = true;
```

Comment:
Should it be inside #else case???

Reply:
Thank you for spotting this, I did miss the #else case!